.github/workflows/editing-toolkit-plugin.yml

```diff
@@ -212,3 +215,43 @@ jobs:
       # This will fail if the textdomain has not been changed to "full-site-editing", which indicates an issue with the sync or build scripts.
       - name: Execute phpcs on newspack-blocks PHP files.
         run: ./vendor/bin/phpcs --standard=apps/editing-toolkit/bin/newspack-block-sync-phpcs.xml
+
+  create-release-and-attach-asset:
+    name: Create release draft and attach asset
+    needs: build
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/editing-toolkit@')
+    steps:
+      - name: Set Release Version
+        id: get_release_version
+        run: echo ::set-output name=version::$(echo $GITHUB_REF | cut -d / -f 3 | sed s/^editing-toolkit@// | sed 's/-rc./ RC/' )
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ steps.get_release_version.outputs.version }}
+          draft: true
+
+      - name: Get build
+        uses: actions/download-artifact@HEAD
+        with:
+          name: editing-toolkit-build-archive
+
+      - name: Build zip archive
+        run: |
+          zip -r editing-toolkit.zip ./*
```

Comment:
The `editing-toolkit-build-archive` artifact should already be a plugin .zip file. I wonder if there's a way to avoid re-zipping it 🤔 Maybe release asset is able to point at a build artifact?

Reply:
'fraid not, seems like a known limitation: https://github.com/actions/upload-artifact/tree/928d1a16d908842c3b25f3eae36ec383fa02e15c#zipped-artifact-downloads

(_tl;dr_ The artifact is not "really" a zip -- internally it's treated as a collection of files and folders by the `upload-artifact` and `download-artifact` actions. It's just the UI that wraps them into a zip for download.)

Reply:
Ah, that makes sense!